.ci/Jenkinsfile

```diff
@@ -383,7 +383,7 @@ pipeline {
       environment {
         HOME = "${env.WORKSPACE}"
         RESULT_FILE = 'apm-agent-benchmark-results.json'
-        NODE_VERSION = '14'
+        BENCH_NODE_VERSION = '14'
```

Comment:
REVIEW NOTE: Rename to `BENCH_NODE_VERSION` to differentiate from other unrelated usages of `NODE_VERSION` in this and other files in the repo.